laser_encoders/test_models_initialization.py

```diff
@@ -0,0 +1,58 @@
+import os
+import tempfile
+
+import pytest
+
+from laser_encoders.download_models import LaserModelDownloader
+from laser_encoders.language_list import LASER2_LANGUAGE, LASER3_LANGUAGE
+from laser_encoders.laser_tokenizer import initialize_tokenizer
+from laser_encoders.models import initialize_encoder
+
+
+def test_validate_achnese_models_and_tokenize_laser3(lang="acehnese"):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        print(f"Created temporary directory for {lang}", tmp_dir)
+
+        downloader = LaserModelDownloader(model_dir=tmp_dir)
+        downloader.download_laser3(lang)
+        encoder = initialize_encoder(lang, model_dir=tmp_dir)
+        tokenizer = initialize_tokenizer(lang, model_dir=tmp_dir)
+
+        # Test tokenization with a sample sentence
+        tokenized = tokenizer.tokenize("This is a sample sentence.")
+
+    print(f"{lang} model validated successfully")
+
+
+def test_validate_english_models_and_tokenize_laser2(lang="english"):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        print(f"Created temporary directory for {lang}", tmp_dir)
+
+        downloader = LaserModelDownloader(model_dir=tmp_dir)
+        downloader.download_laser2()
+
+        encoder = initialize_encoder(lang, model_dir=tmp_dir)
+        tokenizer = initialize_tokenizer(lang, model_dir=tmp_dir)
+
+        # Test tokenization with a sample sentence
+        tokenized = tokenizer.tokenize("This is a sample sentence.")
+
+    print(f"{lang} model validated successfully")
+
+
+def test_validate_kashmiri_models_and_tokenize_laser3(lang="kas"):
+    with tempfile.TemporaryDirectory() as tmp_dir:
+        print(f"Created temporary directory for {lang}", tmp_dir)
+
+        downloader = LaserModelDownloader(model_dir=tmp_dir)
+        lang_codes = LASER3_LANGUAGE[lang]
+
+        for code in lang_codes:
```

Comment:
I think, in this test we should try downloading the model directly for `kas` and expect the ValueError. 
The ability to load a model for a correctly specified language has already been already tested above with Acehnese, so we don't have to test it here once more. 
So in this test, we should check the ability to raise a meaningful error when a language is ambiguous.

Reply:
@avidale is an value error like this correct?

![Screenshot from 2023-11-09 18-38-19](https://github.com/facebookresearch/LASER/assets/91743459/1ff283ae-8a43-4274-8cb2-a24184444d41)

Reply:
I have changed the code [here](https://github.com/facebookresearch/LASER/pull/257/commits/9ec012f171abacaf7e79a32ac5d9142eb00e9944).
It return the value error hence failing the pytest [here](https://github.com/NIXBLACK11/LASER-fork/actions/runs/6812342872/job/18524472837#step:6:54).

Reply:
> It return the value error hence failing the pytest [here](https://github.com/NIXBLACK11/LASER-fork/actions/runs/6812342872/job/18524472837#step:6:54).

Now can you please use `with pytest.raises` to make the test PASS if the ValueError is raised, and FAIL, if it is not?
https://docs.pytest.org/en/7.1.x/how-to/assert.html#assertions-about-expected-exceptions

Reply:
@avidale Thanks, I made the changes now all the tests pass.

Reply:
@avidale I had a question, should I add this in the slow tests too?